README.md

```diff
@@ -1 +1,65 @@
-# BloomDiscordBot
\ No newline at end of file
+# BloomDiscordBot
+
+# Help:
+
+You can type ```$bot_help``` to get details about what commands can be used, along with a brief description of them
+
+# Events:
+
+When a person creates an event within Blooms Discord server, there will initially be a one hour delay from the time of creation to when the details of the event are posted. This will allow the person sometime to make changes to the details, timing and so on.
```

Comment:
This does not match or seemingly not match witht he only 30mins delay i found and commented on

Reply:
> This does not match or seemingly not match witht he only 30mins delay i found and commented on

Yeah, I originally had this as an hour. But then I also had the check for upcoming events within the next 24 hours set to the top of an hour so I changed to 30 minutes so that there wasn't two things running around the same time.

I will update this to reflect that change